.github/workflows/ci.yaml

```diff
@@ -30,31 +30,22 @@ defaults:
 jobs:
   tests:
     runs-on: ${{ matrix.os }}
-    name: "💻-${{matrix.os }} 🐍-${{ matrix.python-version }} 🗃️${{ matrix.pydantic-version }} oechem: ${{ matrix.openeye }}"
+    name: "💻-${{matrix.os }} 🐍-${{ matrix.python-version }} oechem: ${{ matrix.openeye }}"
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest"]
-        pydantic-version: [">1"]
+        os: ["ubuntu-latest", "macos-latest"]
         python-version:
           - "3.10"
           - "3.11"
           - "3.12"
         openeye: ["no"]
         include:
-          - os: "macos-latest"
-            python-version: "3.12"
-            pydantic-version: ">1"
-          - os: "ubuntu-latest"
-            python-version: "3.11"
-            pydantic-version: "<2"
           - os: "ubuntu-latest"
             python-version: "3.11"
-            pydantic-version: ">1"
             openeye: "yes"
           - os: "macos-latest"
             python-version: "3.12"
-            pydantic-version: ">1"
             omff-version: ">0.13"
```

Comment:
what is this?

Reply:
okay, i've removed that and replaced it with a macos w/ openeye check. do you agree with that?